Can't build getsentry/develop on Mac M1 (arm64)

```
matt@XHFHLG2CJP develop % yarn run init
yarn run v1.22.19
$ cp .env.example .env.development && yarn install
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > bootstrap@4.3.1" has unmet peer dependency "jquery@1.9.1 - 3".
warning " > bootstrap@4.3.1" has unmet peer dependency "popper.js@^1.14.7".
warning "babel-preset-gatsby > @babel/plugin-proposal-class-properties@7.17.12" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-preset-gatsby > @babel/plugin-proposal-nullish-coalescing-operator@7.17.12" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-preset-gatsby > @babel/plugin-proposal-optional-chaining@7.17.12" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-preset-gatsby > @babel/plugin-syntax-dynamic-import@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-preset-gatsby > @babel/plugin-transform-runtime@7.18.2" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-preset-gatsby > @babel/plugin-transform-spread@7.17.12" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-preset-gatsby-package > @babel/plugin-transform-typescript@7.18.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > gatsby-plugin-sass@2.8.0" has unmet peer dependency "node-sass@^4.9.0".
warning "gatsby-plugin-sass > sass-loader@7.3.1" has unmet peer dependency "webpack@^3.0.0 || ^4.0.0".
warning " > gatsby-transformer-remark@5.25.1" has incorrect peer dependency "gatsby@^4.0.0-next".
warning " > babel-jest@26.6.3" has unmet peer dependency "@babel/core@^7.0.0".
warning "babel-jest > babel-preset-jest@26.6.2" has unmet peer dependency "@babel/core@^7.0.0".
warning "babel-jest > babel-preset-jest > babel-preset-current-node-syntax@1.0.1" has unmet peer dependency "@babel/core@^7.0.0".
warning "babel-jest > babel-preset-jest > babel-preset-current-node-syntax > @babel/plugin-syntax-bigint@7.8.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-jest > babel-preset-jest > babel-preset-current-node-syntax > @babel/plugin-syntax-import-meta@7.10.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > babel-preset-gatsby@0.4.12" has unmet peer dependency "@babel/core@^7.0.0".
warning " > babel-preset-gatsby-package@0.4.6" has unmet peer dependency "@babel/core@^7.0.0".
warning "babel-preset-gatsby-package > @babel/preset-flow@7.17.12" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-preset-gatsby-package > @babel/preset-flow > @babel/plugin-transform-flow-strip-types@7.17.12" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "babel-preset-gatsby-package > @babel/preset-flow > @babel/plugin-transform-flow-strip-types > @babel/plugin-syntax-flow@7.17.12" has unmet peer dependency "@babel/core@^7.0.0-0".
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
[4/4] 🔨  Building fresh packages...
[-/16] ⠂ waiting...
[-/16] ⠂ waiting...
[-/16] ⠂ waiting...
[11/16] ⠂ sharp
error /Users/matt/Code/getsentry/develop/node_modules/sharp: Command failed.
Exit code: 1
Command: (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
Arguments: 
Directory: /Users/matt/Code/getsentry/develop/node_modules/sharp
Output:
info sharp Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.10.5/libvips-8.10.5-darwin-arm64v8.tar.br
ERR! sharp Prebuilt libvips 8.10.5 binaries are not yet available for darwin-arm64v8
info sharp Attempting to build from source via node-gyp but this may fail due to the above error
info sharp Please see https://sharp.pixelplumbing.com/install for required dependencies
gyp info it worked if it ends with ok
gyp info using node-gyp@9.0.0
gyp info using node@16.16.0 | darwin | arm64
gyp info find Python using Python version 3.11.2 found at "/Library/Frameworks/Python.framework/Versions/3.11/bin/python3"
gyp info spawn /Library/Frameworks/Python.framework/Versions/3.11/bin/python3
gyp info spawn args [
gyp info spawn args   '/Users/matt/.volta/tools/image/node/16.16.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/matt/Code/getsentry/develop/node_modules/sharp/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/matt/.volta/tools/image/node/16.16.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/matt/Library/Caches/node-gyp/16.16.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/matt/Library/Caches/node-gyp/16.16.0',
gyp info spawn args   '-Dnode_gyp_dir=/Users/matt/.volta/tools/image/node/16.16.0/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/matt/Library/Caches/node-gyp/16.16.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/matt/Code/getsentry/develop/node_modules/sharp',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CC(target) Release/obj.target/nothing/../node-addon-api/nothing.o
  LIBTOOL-STATIC Release/nothing.a
warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: Release/nothing.a the table of contents is empty (no object file members in the library define global symbols)
  TOUCH Release/obj.target/libvips-cpp.stamp
  CXX(target) Release/obj.target/sharp/src/common.o
../src/common.cc:24:10: fatal error: 'vips/vips8' file not found
#include <vips/vips8>
         ^~~~~~~~~~~~
1 error generated.
make: *** [Release/obj.target/sharp/src/common.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/matt/.volta/tools/image/node/16.16.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (node:events:527:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
gyp ERR! System Darwin 22.3.0
gyp ERR! command "/Users/matt/.volta/tools/image/node/16.16.0/bin/node" "/Users/matt/.volta/tools/image/node/16.16.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/matt/Code/getsentry/develop/node_modules/sharp
error Command failed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -78,6 +78,9 @@
     "prettier": "2.7.1",
     "react-test-renderer": "^16.13.1"
   },
+  "resolutions": {
+    "*/sharp": "^0.32.0"
+  },
   "volta": {
     "node": "16.16.0",
     "yarn": "1.22.5"

